config/metrics/add-metrics-server-components.yml

```diff
@@ -0,0 +1,156 @@
+#@ load("@ytt:data", "data")
+
+#@ if data.values.add_metrics_server:
```

Comment:
Nice. I think we also have some docs to update for this one.

Reply:
We didn't attempt to update docs or CI to reflect the changes in this PR yet. We just wanted to get our thoughts down "on paper" for others to review.

Reply:
Everything should be up to date now.